bgpd/bgp_rpki.c

```diff
@@ -1656,6 +1713,46 @@ DEFPY (show_rpki_cache_connection,
 	return CMD_SUCCESS;
 }
 
+DEFPY(show_rpki_configuration, show_rpki_configuration_cmd,
+      "show rpki configuration [json$uj]",
+      SHOW_STR RPKI_OUTPUT_STRING
+      "Show RPKI configuration\n"
+      JSON_STR)
+{
+	struct json_object *json = NULL;
+
+	if (uj) {
+		json = json_object_new_object();
+		json_object_boolean_add(json, "enabled",
+					!!listcount(cache_list));
+		json_object_int_add(json, "serversCount", listcount(cache_list));
```

Comment:
Would be handy to list a cache server that is connected too.

Reply:
You can do `show rpki cache-connection`

The command is for displaying the configured timers